modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -0,0 +1,9 @@
+[id="creating-a-new-client-in-keycloak_{context}"]
+= Creating a new client in Keycloak
+
+Create a new client in Keycloak for security, easy identity, and access management. Keycloak acts as an identity broker between Argo CD and OpenShift; as a result, you can login into Argo CD using the OpenShift credentials.
+
+.Procedure
+
+. Configure Keycloak by editing and updating the Argo CD `.sso.spec.provider: keycloak`.
```

Comment:
My bad for not noting this before. But for single step procedures, they want you to use a bullet point, not a number. That's the only thing I'd change in all these docs. They all look great!